app/notifiers/nil_notifier.rb

```diff
@@ -0,0 +1,4 @@
+class NilNotifier
+  def self.call(**args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `call(*)` if you want the method to accept any arguments but don't care about them.